src/externals/crypto/aes.ts

```diff
@@ -103,10 +99,11 @@ export class AesService {
     );
 
     // AES 256 GCM Mode
-    const decipher = crypto.createDecipheriv('aes-256-gcm', key, iv);
+    const decipher: any = crypto.createDecipheriv('aes-256-gcm', key, iv);
```

Comment:
Avoid using `any`, crypto has the Decipher type, which is in fact automatically typed. Therefore this `any` could be removed directly.